ModelReplacementAPI/BodyReplacementBase.cs

```diff
@@ -465,12 +474,14 @@ private IEnumerator WaitForDanceNumberChange()
         #region MirrorDecor Logic
         private void MirrorPatch()
         {
-            foreach (var item in replacementModel.GetComponentsInChildren<Renderer>())
+            if (localPlayer) // Only set our model to different layer, we still want to see others' models!
             {
-                item.shadowCastingMode = ShadowCastingMode.On;
-                item.gameObject.layer = 3;
+                foreach (var item in replacementModel.GetComponentsInChildren<Renderer>())
+                {
+                    item.shadowCastingMode = ShadowCastingMode.On;
+                    item.gameObject.layer = 23;
```

Comment:
Layer 3 can be seen by the main camera, we need to change it to one that it can't see. This is because the model needs to be rendered at all times for the mirror to work, and we don't want to see our own model in front of our camera.